drivers/sensor/lis2ds12/lis2ds12_trigger.c

```diff
@@ -28,8 +24,8 @@ static void lis2ds12_gpio_callback(const struct device *dev,
 
 	ARG_UNUSED(pins);
 
-	gpio_pin_interrupt_configure(data->gpio, cfg->irq_pin,
-				     GPIO_INT_DISABLE);
+	gpio_pin_interrupt_configure_dt(&cfg->gpio_int,
```

Comment:
this function should be checked for errors

Reply:
But the callback returns void in this case. Do you want me to raise a LOG_ERR here?

Reply:
Maybe an assert then?

Reply:
sure